.github/workflows/version-bump.yml

```diff
@@ -0,0 +1,106 @@
+# **what?**
+# This workflow will take a version number and a dry run flag. With that
+# it will run versionbump to update the version number everywhere in the 
+# code base and then generate an update Docker requirements file. If this
+# is a dry run, a draft PR will open with the changes. If this isn't a dry
+# run, the changes will be committed to the branch this is run on.
+
+# **why?**
+# This is to aid in releasing dbt and making sure we have updated 
+# the versions and Docker requirements in all places.
+
+# **when?**
+# This is triggered either manually OR 
+# from the repository_dispatch event "version-bump" which is sent from
+# the dbt-release repo Action
+
+name: Version Bump
+
+on:
+  workflow_dispatch:
+  repository_dispatch:
+    types: [version-bump]
+  
+# Default version number and dry run setting in case we have to run this
+# Action manually and not from a repository_dispatch event.
+# These should be updated if that is the case.
+env:
+  version_number_default: '0.22.0'
+  is_dry_run_default: true
```

Comment:
Instead of codifying this, can we just use inputs for the manual workflow dispatch trigger?
https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#example-workflow-configuration

the trigger above would look something like:
```
on:
  workflow_dispatch:
    inputs:
      version_number:
        required: true
      is_dry_run:
        required: false
        default: 'true'
```